olsodb/db/api.py

```diff
@@ -1,14 +1,18 @@
 from oslo_config import cfg
 from oslo_db import api as db_api
-
+from oslo_db import options
 
 _BACKEND_MAPPING = {'sqlalchemy': 'olsodb.db.sqlalchemy.api'}
 
 
 IMPL = db_api.DBAPI.from_config(cfg.CONF, backend_mapping=_BACKEND_MAPPING)
 
 
+def get_engine():
+    return IMPL.get_engine()
+
 def configure(conf):
+    options.set_defaults(cfg.CONF)
```

Comment:
The idea was to provide ``conf`` from outside (just couple of things, like connect string and some db options, not full blown oslo.config object), and pass it to the implementation, but yeah, maybe that's the way for make it work. I'll try that on implementation side, so db.api will not be cluttered with the options.